pkg/usage/usage.go

```diff
@@ -196,7 +197,10 @@ func (u *Usage) SetValue(v int64) *Usage {
 func (u *Usage) Build() *Usage {
 	// Default ApplicationID for openebs project is OpenEBS
 	v := NewVersion()
-	v.getVersion(false)
+	err := v.getVersion(false)
```

Comment:
Hi @soniasingla the tests are failing as v is not defined here
```suggestion
	v, err := getVersion(false)
```